src/pages/Committees.js

```diff
@@ -587,4 +641,204 @@ function SportsCommittee() {
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: nlg@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function Festmästeriet () {
+    return (
+        <p>
+            Party people, here is your stuff! The Party Committee (Swedish: Festmästeri) is specialized in organizing funny and wholesome
+            parties and events. Members of the committee can do tasks as diverse as preparing the menu, doing PR (public relations, i.e.
+            advertisement), invent fun competitions for gasques or film gyckels. Join them to maximise the fun of Media's parties.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: festmasteriet@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function FilmCommittee () {
+    return (
+        <p>
+            Interested in films? The Film Committee (Swedish: Filmnämden) produces movies, including a high-effort film project for
+            each reception. They handle every part of film-making, from screen-writing, to filming, post-production, etc. You can see
+            their work on their social networks below.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: filmnamnden@medieteknik.com</li>
+                <li>Instagram: @filmnamnden</li>
+                <li><a href="https://youtube.com/@filmnamndenmedieteknik?si=MhexLexOo8W1N8eO">Youtube</a></li>
+            </ul>
+        </p>
+    )
+}
+
+function PhotoGroup () {
+    return (
+        <p>
+            Do you have an interest in photography? The Photo Group (Swedish: Fotogruppen) is a meeting place for all photography people of
+            the Media chapter. You do not need to have technical knowledge of camera, you can join even if you are only interested in looking
+            at others' pictures. Some members in the committee have extensive technical knowledge, some are more on the art aspect, so
+            there is much you can learn from them!
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: fotogruppen@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function MediaSportsCommittee () {
+    return (
+        <p>
+            Come combine your studies with fresh air, exercising and meeting new friends. The Sports Committee (Swedish: Idrottsnämnden) organizes
+            sport activities for all Media students to enjoy.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: idrottsnamnden@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function Matlaget () {
+    return (
+        <p>
+            Prepare your apron and best cooking ustensils, it's time to cook. The Food Team (Swedish: Matlaget) prepares the food
+            for chapter's meetings, from planning what to cook, to buying the ingredients and finally cooking. The committee is open, i.e. anyone
```

Comment:
*medias chapter meetings

Reply:
I put "Media's chapter meetings" instead, I don't think "Medias" work in English.

Reply:
You're right, my misstake